spec/team_spec.rb

```diff
@@ -36,7 +36,7 @@
     end
 
     it "can count the total number of teams" do
-      expect(Team.count_of_teams).to eq(32)
+      expect(Team.count_of_teams).to eq(192)
```

Comment:
I'm not sure where others stand on this but I'd prefer to keep the test displaying the proper number 32, even if it means 'failing' that test when running rspec/spec.  Cause if you just run the team_spec, this test will currently fail and I think we can talk about why the full spec suite test causes it to fail.